README.md

```diff
@@ -117,7 +117,7 @@ It's a matter of adding the following line at the top of a file:
 
 Then pass the proper `-I` argument to the compiler to add the `src` directory to
 the include paths.<br/>
-Note that users are demanded to correctly setup include directories and
+Note that users are required to correctly setup, including directories and
```

Comment:
Here the meaning is changed. Users have to set up correctly the include directories and the libraries search paths. It's not _setting it up, include directories ..._, but _setting up include directories ..._. I'm not sure you got what I mean. Let me know if it's not clear.

Reply:
ah, got it. Will push an update.